pkgs/development/python-modules/wgnlpy/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, setuptools
+, cryptography
+, pyroute2
+}:
+
+buildPythonPackage rec {
+  pname = "wgnlpy";
+  version = "0.1.5";
+  format = "pyproject";
```

Comment:
Should be `pyproject = true;` now (see https://github.com/NixOS/nixpkgs/commit/39c64e232328e09284c766d2b56f2d680e511d41).

Reply:
I've updated and tested it. Thanks for the hint!